ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -182,11 +155,7 @@ const mstp = (state: AppState): StateProps => {
   }
 }
 
-const mdtp = {
-  setCommunityTemplateToInstall,
-}
-
-export const CommunityTemplatesIndex = connect<StateProps, DispatchProps, {}>(
+export const CommunityTemplatesIndex = connect<StateProps, {}, {}>(
```

Comment:
```suggestion
export const CommunityTemplatesIndex = connect<StateProps>(
```
can just remove those empty types entirely!

Reply:
🤦